src/main/java/Daisy.java

```diff
@@ -0,0 +1,52 @@
+import java.util.Scanner;
+
+public class Daisy {
+    public static void main(String[] args) {
+        final String INTRO_PROMPT = "Good day! This is Daisy.\nAny task for today?";
+        final String EXIT_PROMPT = "Ending prompt received. Remember to keep to the deadlines!";
+        final String LINE_BREAK = "____________________________________";
+        Task[] tasks = new Task[100];
+        Task new_task;
+        int task_no = 0;
+
+
+        System.out.println(LINE_BREAK);
+        System.out.println(INTRO_PROMPT);
+        System.out.println(LINE_BREAK);
+
+        Scanner in = new Scanner(System.in);
+        String command = in.nextLine();
+
```

Comment:
Perhaps its a good idea to use SLAP here! Since your Main is more than 30 LoC, readability maybe affected.